sklearn/utils/__init__.py

```diff
@@ -1264,3 +1266,117 @@ def is_abstract(c):
     # itemgetter is used to ensure the sort does not extend to the 2nd item of
     # the tuple
     return sorted(set(estimators), key=itemgetter(0))
+
+
+def _get_response_values(
+    estimator,
+    X,
+    y_true,
+    response_method,
+    pos_label=None,
+    target_type=None,
+):
+    """Compute the response values of a classifier or a regressor.
+
+    The response values are predictions, one scalar value for each sample in X
+    that depends on the specific choice of `response_method`.
+
+    This helper only accepts multiclass classifiers with the `predict` response
+    method.
```

Comment:
Should there be an error when then target is binary and the response is not `predict`?

Reply:
nop, in this case, we will modify the predictions accordingly. I think the issue is when we have multiclass then we only accept `predict`